config/default.json

```diff
@@ -1,3 +1,3 @@
 {
-  "jwtPrivateKey": ""
+  "jwtPrivateKey": "bookingclone_jwtPrivateKey"
```

Comment:
here we just defining the template for all th settings in the application. so it is empty string.
in custom-env-var we specify mapping between app settings and env variables:
eg: we have there jwtPrivateKey: "someKey"
so when we set in the cmd set someKey=12345 (on windows) then we set 12345 to the jwtPrivateKey